js/src/components/RelatedKeyphrasesModal.js

```diff
@@ -91,7 +91,7 @@ class RelatedKeyphrasesModal extends Component {
 							<h2>Modal debug info</h2>
 							<p>
 								The location is: { location }<br />
-								The current database is: { currentDatabase }
+								The current database country is: { countryCode }
```

Comment:
I know it's just debugging info that will go away, but while the test is OK and the country code changes in the state and in the database, here it's always `us`, as if it just takes the value from default?